notebooks/tropical/tropicalgemm.jl

```diff
@@ -155,10 +171,16 @@ md"There are various types of indices"
 VectorizationBase.Index
 
 # ╔═╡ 1afbd730-9742-485e-87d4-bfab8882010e
-md"One can use the `MM` type to load a vectorized data into the SIMD register. For example, To continuously load 4 double precision floating point number (8 bytes) from position 0 into a `Vec`, we can use the following index"
+md"One can use the `MM` type to load a vectorized data into the SIMD register. For example, To continuously load 4 double precision floating point number (8 bytes) from position 1 into a `Vec`, we can use the following index"
 
 # ╔═╡ d0667419-7f6a-4b37-89a4-3b7302e14cec
-vec_index = MM(StaticInt(4), StaticInt(0), StaticInt(8))
+vec_index = MM{4}(1)
+
+# ╔═╡ 3f1ef91e-01cd-4246-93e9-cb4bfbf683de
+begin
```

Comment:
Instead of using begin ... end statement, using let ... end statement is safer because it does not create global variables so that Pluto notebook can be happier.